### ISSUE TITLE
Issue with install.ps1 script

### ISSUE THREAD

**Issue by ZeeshanaliZM** [mandiant/flare-vm#496]
### Description
There seems to be an issue with the powershell script install.ps1.
As required i performed all the prerequisites including unblocking the script, run as admin and setting the execution policy. 

### Steps to Reproduce

**Expected behavior:**
I had downloaded the flare-vm tool before but due to some problems i had to erase the virtual machine and reinstall flare vm.
After running the install.ps1 script normally the script should download chocolatey and then boxstarter.

**Actual behavior:**
When the script tried to download chocolatey, the chocolatey folder was created in ProgramData however the folder is empty. The content of the folder was missing.
Also the script generated an error:
A previous version of chocolatey has been detected and chocolatey fails to install.
I tried the process on a number of systems but the problems still persists.
IN short: chocolatey directory is created but the content of chocolatey fail to install.
I tried the process a no of times by erasing the entire VM but the problem still persists. The problem has only started since last week.

**Comment by n1k0m0** [comment]
Hi ZeeshanaliZM,
I am right now installing Flare VM on a virtual Windows 10 machine and ran into the same problem.
I could successfully circumvent the problem by manually installing chocolatey prior the Flare VM installation as described in https://chocolatey.org/install
After chocolateley installation, you need to start a new powershell session since it overwrites policies for the process.

Greetings,
Nils

**Comment by ZeeshanaliZM** [comment]
Hello Nils,
Thank you for the solution i tried it and it worked out. Thanks a lot.

**Comment by ZeeshanaliZM** [comment]
Hello Nils,
Thank you for the solution i tried it and it worked out. Thanks a lot.
